src/modules/meetings/ui/components/meetings-list-header.tsx

```
@@ -1,12 +1,29 @@
+import { StatusFilter } from "./status-filter"
+import { AgentIdFilter } from "./agent-id-filter"
+import { useMeetingsFilters } from "../../hooks/use-meetings-filters"
+import { ScrollBar , ScrollArea } from "@/components/ui/scroll-area"
+
+export const MeetingsListHeader = () => {
+
+    const [filters, setFilters] = useMeetingsFilters();
 
-export const MettingsListHeader = () => {
     const [isDialogOpen, setIsDialogOpen] = useState(false);
+    const isAnyFilterModified = !!!filters.search || !!!filters.status || !!!filters.agentId;
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)